src/ConfigWidgets/SymbolsDialog.cpp

```diff
@@ -25,6 +25,10 @@
 
 constexpr const char* kFileDialogSavedDirectoryKey = "symbols_file_dialog_saved_directory";
 constexpr const char* kModuleHeadlineLabel = "Add Symbols for <font color=\"#E64646\">%1</font>";
+constexpr const char* kOverrideWarningText =
+    "The build ID in the file you selected does not match. This may lead to unexpected behavior in "
+    "Orbit.<br />If you are sure this is the right file, then you can override to use the file "
+    "with the mismatched build ID.";
```

Comment:
I feel `unexpected behaviour` is too vague. Can we somehow make this more clear?

Reply:
I am going to take this offline